rust/noosphere-into/src/into/html/sphere.rs

```diff
@@ -215,9 +215,9 @@ pub mod tests {
 
         let html = std::str::from_utf8(&bytes).unwrap();
 
-        println!();
-        println!("{}", html);
-        println!();
+        info!("");
+        info!("{}", html);
+        info!("");
```

Comment:
~Could get weird with other formatters? Could `"\n{}\n"` work?~ nevermind this is a test